app/src/renderer/vuex/modules/user.js

```diff
@@ -62,41 +63,42 @@ export default ({ commit, node }) => {
     // to create a temporary seed phrase, we create a junk account with name 'trunk' for now
     async createSeed ({ commit }) {
       let JUNK_ACCOUNT_NAME = 'trunk'
+      let TRUNK_PASSWORD = '1234567890'
```

Comment:
We need to get rid of this. A malicious user could access the last generated trunk account with the standard password.

Reply:
Prerequesit: https://github.com/cosmos/cosmos-sdk/issues/301

Reply:
do you mean in this PR or an another PR? this is not new — this is currently how it works...

Reply:
Just a note. I commented on that, because I implemented it like that and wanted to share the thought.